SIV_scraper

```diff
@@ -0,0 +1,215 @@
+setwd("L:/Prices/Dashboards/SIV")
+
+library(zoo)
+library(dplyr)
+library(readxl)
+library(xlsx)
+library(ggplot2)
+library(openxlsx)
+library(data.table)
+library(xml2)
+library(XML)
+library(purrr)
+library(readr)
+library(stringr)
+library(reshape2)
+library(data.table)
+library(pdftools)
+library(shiny)
+library(writexl)
+library(DT)
+library(tidyr)
+library(tibble)
+library(tabulizer)
+library(rvest)
+library(pdftools)
+library(shinythemes)
```

Comment:
There are a lot of libraries in use here, can you tidy it up and remove redundant ones?